docs/quantize/quantize_example.py

```diff
@@ -0,0 +1,104 @@
+from functools import partial
+
+import tensorflow as tf
+
+from blueoil.networks.<TASK_NAME>.<NETWORK_NAME> import FooNetwork
+
+
+class FooNetworkQuantize(FooNetwork):
```

Comment:
In example code, `FooNetworkQuantize` should inherit [`Base` ](https://github.com/blue-oil/blueoil/blob/master/blueoil/networks/classification/base.py). Not `FooNetwork`.
We assume the document user want to write their network from scratch.

Reply:
I think we need `FooNetwork` which inherit from `Base`.
Then `FooNetwotkQuantize` inherit from `FooNetwork`.

I will add the FooNetwork code in this file.

Reply:
I'm not sure why FooNetwotkQuantize can not directory inherit Base?